### DISCUSSION TITLE
How to disable revalidation?

### DISCUSSION THREAD

**Discussion by BrunoQuaresma** [vercel/swr#469] [General] [answered]
I have a current component with a modal but every time a click in the modal and close it the fetch is being called. I'm currently using this without success:
```
revalidateOnFocus: false,
revalidateOnMount: false
```
Is that possible to say to useSWR "Just loads this data once"?

**Comment by sergiodxa** [comment, marked as answer]
Check #450, there you can see how to use it, however it's not recommended to disable revalidation.
